weatherAPI/models/user.go

```diff
@@ -0,0 +1,86 @@
+package models
```

Comment:
same here.
all this stuff can be deleted, is part of the boilerplate.

Reply:
Deleted